--- auth-ldap-1.0.3/src/Makefile.in~	2006-01-21 01:06:23.000000000 +0200
+++ auth-ldap-1.0.3/src/Makefile.in	2006-01-21 01:51:06.000000000 +0200
@@ -24,7 +24,7 @@
 	./test $(top_srcdir)/auth-ldap.conf
 
 install:: openvpn-auth-ldap.so
-	$(INSTALL) -m 755 openvpn-auth-ldap.so $(DESTDIR)$(prefix)/lib/
+	$(INSTALL) -m 755 openvpn-auth-ldap.so $(DESTDIR)$(libdir)
 
 clean::
 	rm -f $(TEST_OBJS) $(MODULE_OBJS) openvpn-auth-ldap.so test
